dali/base/dasess.cpp

```diff
@@ -2014,6 +2014,10 @@ class CUserDescriptor: implements IUserDescriptor, public CInterface
     StringBuffer signature;//user's digital Signature
 public:
     IMPLEMENT_IINTERFACE;
+    CUserDescriptor()
+    {
+        sessionToken = 0;
```

Comment:
Our preferred style is to initialize on the declaration (i.e. on line 2013). But I'll merge as-is.